extensions/blocks/membership-button/view.scss

```diff
@@ -0,0 +1,50 @@
+/* Additional styling to thickbox that displays modal */
+/* stylelint-disable selector-max-id */
+
+.jetpack-memberships-modal #TB_title {
+	border-radius: 4px 4px 0 0;
+}
+#TB_window.jetpack-memberships-modal {
+	border-radius: 4px;
+	background-color: #f9f9f9;
+	background-image: url( 'https://s0.wp.com/i/loading/loading-64.gif' );
```

Comment:
Any change we could use Gutenberg [Spinner](https://github.com/WordPress/gutenberg/tree/6c9a9b22312acde7e64462f554fe3b9006becc9f/packages/components/src/spinner) component instead?

Reply:
a) That would be impossible with thickbox
b) This is a trick, because this is really under a transparent iframe. Using anything that is not a gif or css-only background would require sorcery with z-indexes.